test/test-project/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "description": "Test project. More info in the readme.",
+    "scripts": {
+        "test": "npm run test:cleanup && jest --detectOpenHandles",
+        "test:cleanup": "rimraf \"../temp\""
+    },
+    "dependencies": {
+        "@asyncapi/generator": "file:../..",
+        "@asyncapi/html-template": "0.16.0"
+    },
+    "devDependencies": {
+        "jest": "^25.5.0",
+        "fs-extra": "^9.1.0"
+    }
```

Comment:
Shouldn't this include `rimraf` as a dev dependency?

Reply:
it should, done 🙇🏼